### ISSUE TITLE
Change log file entries to include human readable timestamp

### ISSUE THREAD

**Issue by julie777** [stevencohn/OneMore#894]
**Is your feature request related to a problem? Please describe.**

I just submitted another issue and had to infer the related part of the log file because the timestamp is not human readable.
(I'm sure you don't want the entire log file in each bug report. :-) )

**Describe the solution you'd like**

Use a human readable format for timestamp on log entries

**Describe alternatives you've considered**
deciphering the timestamp and creating a small program that I can run just to show the timestamp in readable form so that I can extract the useful part of the log.

**Comment by stevencohn** [comment]
In any system with _workflow_ or long-running processes with changing state, timestamps are critical. But OneMore commands are all basically atomic operations and _should_ complete in milliseconds if not one or two seconds. Beyond that, there's not much use.

**Comment by julie777** [comment]
I am not sure I was being clear. Assuming my log file has been running for months and I use OneMore commands frequently every day a quick glance doesn't show me what I just did that I want to include in an issue. I made a guess earlier today and just grabbed what looked like close together timestamps, even though I didn't know what the actual timestamp meant.

I might also be confused because it wasn't about running commands, as I was trying to report an update problem.
